### ISSUE TITLE
Node selection should be indicated for screen readers 

### ISSUE THREAD

**Issue by plannero** [umbraco/Umbraco-CMS#14520]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.4.0

### Bug summary

In the ”move page” panel, when you select a destination node, the selection is visually indicated with a “check” symbol. But for non-sighted users this information is not available at all. 

### Specifics

You should set `aria-selected` or `aria-checked` attribute (depending on element `role`) for the node at the same time as toggling the symbol.
<img width="299" alt="node-selected" src="https://github.com/umbraco/Umbraco-CMS/assets/2157085/ac696729-a787-447c-8af0-cca8d8cacd2e">

### Steps to reproduce

This problem appears when you want to move a node in the content tree, as you select a destination (parent node), while using a screen reader. 

### Expected result / actual result

After selecting a parent node as destination, the selected node should be announce as selected by the screen reader, so that you know you did a proper selection before saving changes.

**Comment by elit0451** [comment]
Thank you for raising this concern @plannero 🙌 

We encourage you to submit a PR with your suggestion! I will mark this as up for grabs in case you or anyone else wants to help us fix this. 🙂
